pkg/vm/vm.go

```diff
@@ -918,7 +918,7 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 	case opcode.POW:
 		exp := v.estack.Pop().BigInt()
 		a := v.estack.Pop().BigInt()
-		if ei := exp.Int64(); !exp.IsInt64() || ei > math.MaxInt32 || ei < 0 {
+		if ei := exp.Int64(); !exp.IsInt64() || ei > maxSHLArg || ei < 0 {
```

Comment:
We may also adjust the test for `max+1` value.

Reply:
Fixed.